tests/manager/api/controller/test_work.py

```diff
@@ -68,7 +68,6 @@ def work_fixture(
 class TestWorkController:
     def test_contributor(self, work_fixture: WorkFixture):
         m = work_fixture.manager.work_controller.contributor
-        work_fixture.collection.data_source = None
```

Comment:
This is actually invalid, because datasource cannot be null. I think we set this previously in order to reduce the number of facets coming back. Doing this here causes the configuration to error because datasource is required. 

So I removed this and adjusted for the extra facets having a datasource generates.